.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: ""
+labels: ["bug"]
+assignees: ''
+---
+
+#### Describe the bug
+<!-- A clear and concise description of what the bug is. -->
+
+#### To Reproduce
+<!-- Steps that can be taken to reproduce the behaviour -->
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+#### Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+#### Screenshots
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+#### Logs
+<!-- If applicable, add logs to help explain your problem. -->
+
+#### Environment
+
+- Software version(if relevant): <!-- e.g. 1.2 -->
+- Cloud Environment: <!-- e.g. GKE -->
+- Kubernetes version (output from `kubectl version --short`):
```

Comment:
Those don't make as much sense for this project, I think. We could generalize to environment information.

Reply:
Agreed, changed